src/x_connective.c

```diff
@@ -1581,15 +1589,46 @@ int value_getfloat(t_symbol *s, t_float *f)
 int value_setfloat(t_symbol *s, t_float f)
 {
     t_vcommon *c = (t_vcommon *)pd_findbyclass(s, vcommon_class);
-    if (!c)
+    if (!c || c->c_atom.a_type != A_FLOAT)
```

Comment:
I think `value_setfloat` and `value_setsymbol` only should return 1 if the value object doesn't exist. If they value is a symbol, we can make it a float and vice verca. So the code should read:
```
if (!c)
    return (1);
SETFLOAT(&c->c_atom, f);
return (0);
```

Reply:
Done.